node/declaration.test.ts

```diff
@@ -75,3 +77,53 @@ test('In source config works independent of the netlify.toml file if a path is d
   const declarationsWithoutISCPath = getDeclarationsFromConfig(tomlConfig, funcConfigWithoutPath, deployConfig)
   expect(declarationsWithoutISCPath).toEqual(expectedDeclarationsWithoutISCPath)
 })
+
+test('In source config works if only the cache config property is set', () => {
+  const tomlConfig = [{ function: 'geolocation', path: '/geo', cache: 'off' }]
+
+  const funcConfig = {
+    geolocation: { cache: 'manual' },
+  } as Record<string, FunctionConfig>
+
+  const expectedDeclarations = [{ function: 'geolocation', path: '/geo', cache: 'manual' }]
+
+  expect(getDeclarationsFromConfig(tomlConfig, funcConfig, deployConfig)).toEqual(expectedDeclarations)
+})
+
+test("In source config path property works if it's not an array", () => {
+  const tomlConfig = [{ function: 'json', path: '/json-toml', cache: 'off' }]
+
+  const funcConfig = {
+    json: { path: '/json', cache: 'manual' },
+  } as Record<string, FunctionConfig>
```

Comment:
Do you need the `as` here?

Reply:
Sorry, my comment came in late, can you explain you wondering its necessity?

Reply:
@eduardoboucas do you still want me to address this one?

Reply:
Sorry, I had missed your question. I thought the type could be inferred, but I can see that's not the case. Ignore me!